### ISSUE TITLE
Latest git version doesn't work anymore

### ISSUE THREAD

**Issue by maxigaz** [Jazqa/kwin-quarter-tiling#27]
Using Manjaro Linux (64 bit) with Plasma 5.10.4, the latest git version installed through the AUR package no longer works for me. I've tried going to the system settings, KWin scripts and then disable and re-enable the script, also using the default settings, but I get the same results.

However, I don't have this issue after downgrading to an earlier version of the AUR package (which is shown up as `kwin-scripts-quarter-tiling-git-1:r141.4b70a9e-1-any.pkg.tar.xz` when using `downgrade`).

**Comment by Jazqa** [comment]
Can you approximately estimate the last time you updated? None of the commits have been very big and it would help me locate the problem. As an unrelated note, I need to install an Arch on a virtual machine as so many of you use the AUR package.

EDIT: Dumb me. You did specify a version which works. I'll track this down as soon as possible.

**Comment by Jazqa** [comment]
Sorry for this. Major screw-up from my part. After changing the `Ignored Captions` setting to ignore all windows containing the entries as a part of their caption (#25), leaving the setting empty ended up ignoring all windows with a caption.

Thanks for filing the issue. Would have taken me a while to notice this issue as I have a few window captions ignored. I thought I tested the feature well but I only tested a bunch of different entries, didn't even think about testing the feature with no entries whatsoever.
